### ISSUE TITLE
UI selects Tab 2 by default on navigation to example plugin page 

### ISSUE THREAD

**Issue by n3wscott** [vmware-tanzu/octant#179]
UI defaults to selecting `Tab 2` on example plugin after installing with `make install-test-plugin` and restarting `octant`.

**Comment by wwitzel3** [comment]
We need to recheck with the latest Octant and example plugin to see if this is still an issue.

**Comment by wwitzel3** [comment]
Even though `SetAccessor` is being called, the JSON in the stream does not contain an `accessor`.

```0: {metadata: {type: "cardList", title: [{metadata: {type: "text"}, config: {value: "Tab 1"}}]},…}
config: {cards: [,…]}
metadata: {type: "cardList", title: [{metadata: {type: "text"}, config: {value: "Tab 1"}}]}
title: [{metadata: {type: "text"}, config: {value: "Tab 1"}}]
type: "cardList"```

This causes the `accessor` to not be set by the `ngOnChanges` in the tabs component: https://github.com/vmware-tanzu/octant/blob/master/web/src/app/modules/overview/components/tabs/tabs.component.ts#L55

As a result, the `setMarker` code receives a `nil` accessor, which causes undefined behavior, in this case, the last tab in the list being selected as active. https://github.com/vmware-tanzu/octant/blob/master/web/src/app/modules/overview/components/tabs/tabs.component.ts#L78

**Comment by wwitzel3** [comment]
I attempted to call `SetAccessor` outside of the `gen` function, but still no `accessor` renders on the JSON from the plugin content response.

```
		component1 := gen("Tab 1", "tab1", request.Path)
		component2 := gen("Tab 2", "tab2", request.Path)

		contentResponse := component.NewContentResponse(component.TitleFromString("Example"))
		component1.SetAccessor("tab1")
		component2.SetAccessor("tab2")
```

**Comment by wwitzel3** [comment]
fixed by #369
